### ISSUE TITLE
Auditing Feature

### ISSUE THREAD

**Issue by mmundi72** [danmunn/redmine_dmsf#896]
It would be great to have an auditing feature for administrators which serves the following possibilities:

- Show all documents a specific user has accessed (write/read)
- Show all users who have accessed a specific document (write/read)
- It should be possible to define a custom date range
- It should be possible to export the resultat als CSV-File for an external auditor

**Comment by picman** [comment]
Maybe not fully covered your requirements but some basic functionality is already present:

1) User's activity.
2) Download entries table in the revision details.

**Comment by mmundi72** [comment]
Yes, I know about these functionalities - but it doesn't fully cover what I need (it's just to complicated to get a good overview)
